.github/workflows/e2e.yml

```diff
@@ -53,3 +53,44 @@ jobs:
           ./wait-for-server.sh
       - name: Start Tests
         run: xvfb-run --server-args="-screen 0 1024x768x24" ./gradlew e2eTests
+  E2E-testing-unstable:
```

Comment:
Instead of duplicating, the unstable variant should be defined with the `matrix` option